### PR TITLE
Register attendance through dashboard

### PR DESCRIPTION
**Pull request by NjaalSoerland**
Enables manual registration of attendees for events, through the dashboard

### CODE REVIEW

**Comment by BillySBarrett** [comment]
Holdup jeg kan fikse design snart

**Comment by NjaalSoerland** [comment]
> > Holdup jeg kan fikse design snart
> 
> Pog! Litt nattejobbing fra Billy

Woops æ fiksa det:)

**Comment by BillySBarrett** [comment]
> > > Holdup jeg kan fikse design snart
> > 
> > 
> > Pog! Litt nattejobbing fra Billy
> 
> Woops æ fiksa det:)

woops fiksa bedre

**Comment by BillySBarrett** [comment]
oh no...

**Comment by NjaalSoerland** [comment]
Reverted my recent styling changes due to conflicting commits. Works as is now but think optimally GenericTable should be used for consistency with the rest of the dashboard. It should look something like the "Bedrifter" tab imo

File: apps/dashboard/src/app/(dashboard)/event/[id]/details-attendance.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          return (
+            <Checkbox
+              defaultChecked={attendee.attended}
+              onChange={(event) => toggleAttendance(attendee.userId, attendee.attendanceId, event.target.checked)}
+            />
+          )
+        },
+      }),
+    ],
+    []
```
Comment (junlarsen) [inline apps/dashboard/src/app/(dashboard)/event/[id]/details-attendance.tsx:39]: missing useEffect dependencies

File: apps/dashboard/src/app/(dashboard)/event/[id]/page.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { useEventDetailsContext } from "./provider"
 import { EventCompaniesPage } from "./companies-page"
 import { EventEditCard } from "./edit-card"
 import { useRouter } from "next/navigation"
+import { EventDetailsAttendance } from "./details-attendance"
```
Comment (junlarsen) [inline apps/dashboard/src/app/(dashboard)/event/[id]/page.tsx:10]: small nit, rename to attendance-page to match the other tabs

**Review by junlarsen — CHANGES_REQUESTED** [review]
two small nits otherwise lgtm

File: apps/dashboard/src/app/(dashboard)/event/[id]/page.tsx (hunk 2 above)
Comment (NjaalSoerland) [reply apps/dashboard/src/app/(dashboard)/event/[id]/page.tsx:10]: how so? both the file and import seem to match the naming convention?

Comment (junlarsen) [reply apps/dashboard/src/app/(dashboard)/event/[id]/page.tsx:10]: I was thinking of this one https://github.com/dotkom/monoweb/blob/main/apps/dashboard/src/app/(dashboard)/event/%5Bid%5D/companies-page.tsx

renaming details-attendance to EventAttendancePage

**Review by junlarsen — APPROVED** [review]

**Pull request merged**